source/retryable-reads/tests/aggregate.yml

```diff
@@ -5,6 +5,7 @@ runOn:
     -
         minServerVersion: "4.1.7"
         topology: ["sharded", "load-balanced"]
+        serverless: "forbid"
```

Comment:
`topology: ["single", "replicaset"]` in the other requirement would never match serverless since it'd either identify as sharded or load-balanced, correct?

Reply:
From my understanding, yes.